engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailAggregationProcessor.java

```diff
@@ -75,6 +76,8 @@ public class EmailAggregationProcessor extends SystemEndpointTypeProcessor {
     protected static final String TAG_KEY_APPLICATION = "application";
     protected static final String TAG_KEY_ORG_ID = "orgid";
 
+    public static final String DAILY_DIGEST_QUERY_PARAM_TYPE = "daily_digest";
```

Comment:
Because the trackinq query parameter is a constant, it would be better to add only it in Qute templates only.
Based on what you did, you can just add:
`{@String query_params = 'from=notification_daily_digest'}` as first line of `insightsEmailBodyLight.html` 
`{@String query_params = 'from=notification_instant_email'}` as first line of `insightsEmailBody.html`
Since the java part will remain unchanged.

Reply:
Note: in unit tests, daily digests will have `notification_instant_email` as parameter except for `TestSingleDailyTemplate` which should have the correct value: `notification_daily_digest`.
The 'fix' will come with the removal of deprecated code about daily digest by app.

Reply:
Ok, looked through `TestSingleDailyTemplate` and what you said makes sense, I'll make the change.